packages/patterns/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@endo/patterns",
   "version": "0.1.0",
-  "private": null,
```

Comment:
Please look after commit 1 to see the changes I made after repackage. Did I delete exactly what I should have deleted?

Reply:
All good.